docs/content/manual/v1.8/manual.yml

```diff
@@ -263,8 +263,8 @@ sections:
 
         This option reads in the named file and binds its contents to the given
         global variable.  If you run jq with `--rawfile foo bar`, then `$foo` is
-        available in the program and has a string whose contents are to the texts
-        in the file named `bar`.
+        available in the program and has a string whose contents is set to the
+        texts in the file named `bar`.
```

Comment:
Is this wording in other versions of the manual?

also wondering if it should say ”text” instead ”texts”? 🤔

Reply:
I noticed the missing word in jq(1) (version 1.7.1), but haven't checked elsewhere.

I too thought "text" more appropriate, but I don't know whether multiple strings or lines are meant which is why I left the word as is.